Source/Editor/EditorCore/ResourceOperations.cpp

```diff
@@ -254,6 +254,50 @@ Resource* DuplicateResource(Resource* resource, StringParam expectedNewName)
   return duplicate;
 }
 
+bool MoveResource(Resource* resource, ContentLibrary* targetContentLibrary, ResourceLibrary* targetResourceLibrary)
+{
+    if (PL::gEngine->IsReadOnly())
+    {
+        DoNotifyWarning("Resources", "Cannot rename resources while in read-only mode");
+        return false;
+    }
+
+    ResourceManager* resourceManager = resource->GetManager();
+    String resourceTypeName = resourceManager->mResourceTypeName;
```

Comment:
nitpick: unused variable, can avoid this copy operation

Reply:
This variable is used. It's part of the ResourceEvent that we are sending.

Reply:
I don't see it used there. Am I missing something?

Reply:
I thought you were talking about the resource manager, that's my bad. Removed!